src/datagen/pii/privy/requirements.bazel.txt

```diff
@@ -1356,33 +1356,46 @@ py4j==0.10.9.7 \
+    --hash=sha256:05fe7994745b634c5fb16ce5717e39a1ac1fac3e2b0795232841660aa76647cd \
+    --hash=sha256:06ca79080ef89d6529bb8e5074d4b4f6086143b2520494fcb7cf8a99079cde93 \
+    --hash=sha256:097828b55321897db0e1dbfc606e3ff8101ae5725673498cbfa7754ee0da80e4 \
+    --hash=sha256:0f6f053cb66dc24091f5511e5920e45c83107f954a21032feadc7b9e3a8e7851 \
+    --hash=sha256:11e045dfa09855b6d3e7705a37c42e2dc2c71d608fab34d3c23df2e02df9aec3 \
+    --hash=sha256:1a8ae88c0038d1bc362a682320112ee6774f006134cd5afc291591ee4bc06505 \
+    --hash=sha256:1daab52050a1c48506c029e6fa0944a7b2436334d7e44221c16f6f1b2cc9c510 \
+    --hash=sha256:2a145dab9ed7849fc1101bf03bcdc69913547f10513fdf70fc3ab6c0a50c7eee \
+    --hash=sha256:30d8494870d9916bb53b2a4384948491444741cb9a38253c590e21f836b01222 \
+    --hash=sha256:323cbe60210173ffd7db78bfd50b80bdd792c4c9daca8843ef3cd70b186649db \
+    --hash=sha256:32542164d905002c42dff896efdac79b3bdd7291b1b74aa292fac8450d0e4dcd \
+    --hash=sha256:33c1f6110c386464fd2e5e4ea3624466055bbe681ff185fd6c9daa98f30a3f9a \
+    --hash=sha256:3c76807540989fe8fcd02285dd15e4f2a3da0b09d27781abec3adc265ddbeba1 \
+    --hash=sha256:3f6d5faf4f1b0d5a7f97be987cf9e9f8cd39902611e818fe134588ee99bf0283 \
+    --hash=sha256:450e4605e3c20e558485f9161a79280a61c55efe585d51513c014de9ae8d393f \
+    --hash=sha256:470ae0194fbfdfbf4a6b65b4f9e0f6e1fa0ea5b90c1ee6b65b38aecee53508c8 \
+    --hash=sha256:4756a2b373a28f6166c42711240643fb8bd6322467e9aacabd26b488fa41ec23 \
+    --hash=sha256:58c889851ca33f992ea916b48b8540735055201b177cb0dcf0596a495a667b00 \
+    --hash=sha256:6263cffd0c3721c1e348062997babdf0151301f7353010c9c9a8ed47448f82ab \
+    --hash=sha256:78d4a77a46a7de9388b653af1c4ce539350726cd9af62e0831e4f2bd0c95a2f4 \
+    --hash=sha256:7a8089d7e77d1455d529dbd7cff08898bbb2666ee48bc4085203af1d826a33cc \
+    --hash=sha256:906b0dc25f2be12e95975722f1e60e162437023f490dbd80d0deb7375baf3171 \
+    --hash=sha256:922e8b49b88da8633d6cac0e1b5a690311b6758d6f5d7c2be71acb0f1e14cd61 \
+    --hash=sha256:96d64e5ba7dceb519a955e5eeb5c9adcfd63f73a56aea4722e2cc81364fc567a \
+    --hash=sha256:981670b4ce0110d8dcb3246410a4aabf5714db5d8ea63b15686bce1c914b1f83 \
+    --hash=sha256:a8eeef015ae69d104c4c3117a6011e7e3ecd1abec79dc87fd2fac6e442f666ee \
+    --hash=sha256:b8b3f4fe8d4ec15e1ef9b599b94683c5216adaed78d5cb4c606180546d1e2ee1 \
+    --hash=sha256:be28e1a07f20391bb0b15ea03dcac3aade29fc773c5eb4bee2838e9b2cdde0cb \
+    --hash=sha256:c7331b4ed3401b7ee56f22c980608cf273f0380f77d0f73dd3c185f78f5a6220 \
+    --hash=sha256:cf87e2cec65dd5cf1aa4aba918d523ef56ef95597b545bbaad01e6433851aa10 \
+    --hash=sha256:d0351fecf0e26e152542bc164c22ea2a8e8c682726fce160ce4d459ea802d69c \
+    --hash=sha256:d264ad13605b61959f2ae7c1d25b1a5b8505b112715c961418c8396433f213ad \
+    --hash=sha256:e592e482edd9f1ab32f18cd6a716c45b2c0f2403dc2af782f4e9674952e6dd27 \
+    --hash=sha256:fada8396bc739d958d0b81d291cfd201126ed5e7913cb73de6bc606befc30226
+    # via
+    #   -r requirements.in
```

Comment:
this seems incorrect? isn't pyarrow not in our requirements?

Reply:
Yes you're corect. Fixed in latest commit.